store/postgres/src/relational.rs

```diff
@@ -514,6 +515,32 @@ impl Layout {
         Ok(entities)
     }
 
+    pub fn find_range(
+        &self,
+        conn: &mut PgConnection,
+        entity_type: &EntityType,
+        block_range: Range<BlockNumber>,
+    ) -> Result<BTreeMap<BlockNumber, Vec<Entity>>, StoreError> {
+        let table = self.table_for_entity(entity_type)?;
+        let mut entities: BTreeMap<BlockNumber, Vec<Entity>> = BTreeMap::new();
+        if let Some(vec) = FindRangeQuery::new(table.as_ref(), block_range)
+            .get_results::<EntityData>(conn)
+            .optional()?
+        {
+            for e in vec {
+                let block = e.clone().deserialize_block_number::<Entity>()?;
+                let en = e.deserialize_with_layout::<Entity>(self, None)?;
+                match entities.get_mut(&block) {
```

Comment:
This `match` statement would be more idiomatic as `entities.entry(&block).or_default().push(en)`

Reply:
Beautiful. I remember pondering if it can become one-liner with no luck. Thanks!